subsys/bluetooth/host/audio/chan.c

```diff
@@ -2492,7 +2500,7 @@ int bt_audio_broadcast_sink_stop(struct bt_audio_broadcast_sink *sink)
 		return -EINVAL;
 	}
 
-	chan = sink->chan;
+	chan = &sink->chans[0];
```

Comment:
This works because all channels of a sink have the same state, so it is sufficient to check one of them?

Reply:
All channels have the same state.